source/structured-streaming.txt

```diff
@@ -160,6 +213,74 @@ more about continuous processing, see the `Spark documentation <https://spark.ap
 
    tabs:
 
+     - id: java-sync
+       content: |
+
+         To use continuous processing with the MongoDB Spark Connector, 
+         add the ``trigger()`` method to the ``writeStream`` property 
+         of the streaming Dataset or DataFrame that you create from 
+         your MongoDB read stream. In your ``trigger()``, use the 
+         ``Trigger.Continuous()`` property.
+         
+         .. note:: 
+         
+            The connector populates its read stream from your MongoDB 
+            deployment's change stream. To populate your change stream, 
+            perform update operations on your database.
+         
+            To learn more about change streams, see 
+            :manual:`Change Streams </changeStreams>` in the MongoDB 
+            manual.
+         
+         Specify read stream configuration settings on your local 
+         SparkSession ``readStream``. You must specify the following 
+         configuration settings to read from MongoDB:
+         
+         .. list-table::
+            :header-rows: 1
+            :stub-columns: 1
+            :widths: 10 40
+         
+            * - Setting
+              - Description
+         
+            * - ``readStream.format()``
+              - The format to use for read stream data. Use ``mongodb``.
+         
+            * - ``writeStream.trigger()``
+              - Enables continuous processing for your read stream. 
+                Import and use the ``Trigger.Continuous()`` property.
+
+         The following code snippet shows how to use the preceding 
+         configuration settings to stream data from MongoDB:
+
+         .. code-block:: java
+            :copyable: true
+            :emphasize-lines: 1, 5, 10
+
+            import org.apache.spark.sql.streaming.Trigger;
+ 
+            Dataset<Row> streamingDataset = <local SparkSession>
+              .readStream()
+              .format("mongodb")
+              .load();
+ 
+            DataStreamWriter<Row> dataStreamWriter = streamingDataset
+              .writeStream()
+              .trigger(Trigger.Continuous("1 second"))
+              .format("memory")
+              .outputMode("append");
+ 
+            StreamingQuery query = dataStreamWriter.start();
```

Comment:
Suggestion: highlight this line since it seems important enough to create the following note.